code/game/objects/items/storage/toolbox.dm

```diff
@@ -198,13 +198,15 @@
 	has_latches = FALSE
 	force = 19
 	throwforce = 22
+	var/base_force = 20
+	var/base_throwforce = 23
```

Comment:
why not use initial?
also why does this not match the actual base value of throwforce?

Reply:
Remember how I'm having a weird morning, this is why, because I thought I did this.